app/pudding/tests/test_forms.py

```diff
@@ -0,0 +1,145 @@
+from django.test import TestCase, Client
+
+from django.conf import settings
+from django.utils.translation import activate
+from django.contrib.auth import get_user_model
+
+from .. import forms
+
+
+class RegistrationForm(TestCase):
```

Comment:
не уверен какие соглашение приняты в питоне/django, обычно тестовые классы именуются  `<имя тестируемого класса>Test`
```suggestion
class RegistrationFormTest(TestCase):
```

Reply:
да, я думал, что одинаковые имена - скорее всего не самая лучшая идея, но продолжил из-за разных пространств имен

посмотрел тут:
https://docs.djangoproject.com/en/3.2/intro/tutorial05/
https://docs.djangoproject.com/en/3.2/topics/testing/advanced/


да, называют Test или Tests (в зависимости от кол-ва методов внутри) или XxxxxTestCase
буду дописывать Test(s)